openmc/deplete/microxs.py

```diff
@@ -97,7 +96,15 @@ def from_model(cls,
             run_kwargs.setdefault('cwd', temp_dir)
             statepoint_path = model.run(**run_kwargs)
 
-            with StatePoint(statepoint_path) as sp:
+            # Load summary object -- if model.init_lib() was called, the
+            # summary.h5 file is in the current working directory
+            if model.is_initialized:
+                summary = Summary('summary.h5')
+            else:
+                summary = Summary(Path(temp_dir) / 'summary.h5')
```

Comment:
```suggestion
            # summary.h5 file is in the current working directory
            summary_path = 'summary.h5'
            if model.is_initialized:
                summary = Summary(summary_path)
            else:
                summary_path = (Path(temp_dir) / summary_path).as_posix()
                summary = Summary(summary_path)
```

Reply:
I took a slightly different approach by just adding support for `Path` objects in `Summary`